alf/drivers/policy_driver.py

```diff
@@ -47,18 +46,21 @@ def __init__(self,
             observers (list[Callable]): An optional list of observers that are
                 updated after every step in the environment. Each observer is a
                 callable(time_step.Trajectory).
-            use_rollout_state (bool): Include the RNN state for the experiences
-                used for off-policy training
             metrics (list[TFStepMetric]): An optional list of metrics.
             training (bool): True for training, false for evaluating
             greedy_predict (bool): use greedy action for evaluation (i.e.
                 training==False).
         """
+        assert mode in [
+            "on_policy_training", "predict", "greedy_predict", "rollout"
```

Comment:
`rollout => off_policy_training` seems easier to understand ?

Reply:
updated to constant integer OFF_POLICY_TRAINING